src/modules/add.js

```diff
@@ -0,0 +1,19 @@
+export default function addTodo(array) {
+  const todoText = document.querySelector('#taskInput').value;
+
+  for (let i = 0; i < array.length; i += 1) {
+    array[i].index = i + 1;
+  }
```

Comment:
Hey Ivan, it would be nice if you use an array method like `.forEach()` or `.map()` to itareate over the array and set the index for each element in the array.